servers/audio_server.h

```diff
@@ -129,6 +131,26 @@ class AudioDriver {
 	void reset_profiling_time() { prof_time.set(0); }
 #endif
 
+	// Samples handling.
+	virtual bool is_stream_registered_as_sample(const Ref<AudioStream> &p_stream) const {
+		return false;
+	};
+	virtual void register_sample(const Ref<AudioSample> &p_sample){};
+	virtual void unregister_sample(const Ref<AudioSample> &p_sample){};
+	virtual void start_sample_playback(const Ref<AudioSamplePlayback> &p_playback){};
+	virtual void stop_sample_playback(const Ref<AudioSamplePlayback> &p_playback){};
+	virtual void set_sample_playback_pause(const Ref<AudioSamplePlayback> &p_playback, bool p_paused){};
+	virtual bool is_sample_playback_active(const Ref<AudioSamplePlayback> &p_playback) { return false; };
+	virtual void update_sample_playback(const Ref<AudioSamplePlayback> &p_playback, const StringName &p_bus, float p_pan = 0.0f, float p_volume_db = 0.0f, float p_pitch_scale = 0.0f){};
+
+	virtual void set_sample_bus_count(int p_count){};
+	virtual void remove_sample_bus(int p_bus){};
+	virtual void add_sample_bus(int p_at_pos = -1){};
+	virtual void move_sample_bus(int p_bus, int p_to_pos){};
+	virtual void set_sample_bus_volume_db(int p_bus, float p_volume_db){};
+	virtual void set_sample_bus_solo(int p_bus, bool p_enable){};
+	virtual void set_sample_bus_mute(int p_bus, bool p_enable){};
```

Comment:
```suggestion
	}
	virtual void register_sample(const Ref<AudioSample> &p_sample) {}
	virtual void unregister_sample(const Ref<AudioSample> &p_sample) {}
	virtual void start_sample_playback(const Ref<AudioSamplePlayback> &p_playback) {}
	virtual void stop_sample_playback(const Ref<AudioSamplePlayback> &p_playback) {}
	virtual void set_sample_playback_pause(const Ref<AudioSamplePlayback> &p_playback, bool p_paused) {}
	virtual bool is_sample_playback_active(const Ref<AudioSamplePlayback> &p_playback) { return false; }
	virtual void update_sample_playback(const Ref<AudioSamplePlayback> &p_playback, const StringName &p_bus, float p_pan = 0.0f, float p_volume_db = 0.0f, float p_pitch_scale = 0.0f) {}

	virtual void set_sample_bus_count(int p_count) {}
	virtual void remove_sample_bus(int p_bus) {}
	virtual void add_sample_bus(int p_at_pos = -1) {}
	virtual void move_sample_bus(int p_bus, int p_to_pos) {}
	virtual void set_sample_bus_volume_db(int p_bus, float p_volume_db) {}
	virtual void set_sample_bus_solo(int p_bus, bool p_enable) {}
	virtual void set_sample_bus_mute(int p_bus, bool p_enable) {}
```

Reply:
It's `clang-format` that made me change this.

Reply:
Yes because you added `;` at the end, if you remove them this is correct